### PR TITLE
SP-5018-asg-execute-command

### PR DESCRIPTION
**Pull request by talbe**
### General
- [x] Go over texts with @stackpulse/product
- [x] Explicitly discuss the _playbook name_, _image_ and _badges_
- [x] Explicitly consider various _benefits_. As a rule of thumb, there should be 2-3 of them
- [x] If the playbook has Slack outputs use the Slack experience template

### New playbooks
- [x] Tested Playbook in a production tenant
- [x] Added a link in the [README.MD](https://github.com/stackpulse/playbooks/blob/master/README.md)

### Playbook structure
- [x] For improved human-readability, please add a new empty line between the steps
- [x] Please add a one-liner comment block before each step, for example: `# Retrieve long connections from the database server`
- [x] When considering step IDs, please use short, but descriptive terms, explaining what is being done, such as `retrieve_connections_data` or `update_storage_size`
- [x] When writing params descriptions, start with a capital letter and phrase as a short sentence, such as `User name for database connection` or `Address of the database cluster`

### CODE REVIEW

File: aws/asg-execute-command/README.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
+# AWS Auto Scaling Group Execute Command
+
+<img src="../../images/aws.svg" width="150">
+
+![service](https://img.shields.io/static/v1?label=service&message=AWS&style=flat&logo=AWS&color=FF9900)
+![uses-slack](https://img.shields.io/static/v1?label=uses&message=Slack&style=flat&logo=slack&color=4A154B)
+
+[![Import to StackPulse](../../images/open_in_stackpulse.svg)](https://app.stackpulse.io/playbook/create?tab=playbook#https://github.com/stackpulse/playbooks/blob/master/aws/asg-execute-command/playbook.yaml)
+
+This playbook execute command for given AWS Auto Scaling Group and and verify it's health
```
Comment (hagaishapira) [inline aws/asg-execute-command/README.md:10]:
```suggestion
This playbook executes a command on each instance of an AWS Auto Scaling Group and and then verifies its health.
```

File: aws/asg-execute-command/playbook.yaml [hunk 2]
```diff
@@ -0,0 +1,159 @@
+apiVersion: stackpulse.io/v1
+kind: Playbook
+metadata:
+  description: "Execute command for given AWS Auto Scaling Group and and verify it's health"
```
Comment (hagaishapira) [inline aws/asg-execute-command/playbook.yaml:4]:
```suggestion
  description: "Execute a command on all instances of an AWS Auto Scaling Group and and verify its health"
```

**Comment by hagaishapira** [comment]
@talbe Also please look at the new graphics guidelines, you need to use the template Lihi created and place the outputs there. Please reach out to her if you're having trouble with it.

**Comment by talbe** [comment]
@hagaishapira Done

File: aws/asg-execute-command/playbook.yaml [hunk 3]
```diff
@@ -0,0 +1,159 @@
...
+      $.target_health_foreach.TargetHealthDescriptions[0].TargetHealth.State
+      "healthy"}}'
+    steps:
+      - name: us-docker.pkg.dev/stackpulse/public/slack/message/snippet
+        id: slack-send-snippet
+        env:
+          CONTENT: >
+            Execute command on autoscaling group has finished.
+
+            autoscaling group {{ $.query_chosen_autoscaling_group.AutoScalingGroups[0].AutoScalingGroupName }} is healthy and ready for operation
```
Comment (hagaishapira) [inline aws/asg-execute-command/playbook.yaml:140]: give a meaningful name to the snippet

File: aws/asg-execute-command/playbook.yaml [hunk 4]
```diff
@@ -0,0 +1,159 @@
...
+        envFrom:
+          integrationRef: "{{ $.params.slackIntegration }}"
+  - when: '{{ eq
+      $.target_health_foreach.TargetHealthDescriptions[0].TargetHealth.State
+      "unhealthy"}}'
+    steps:
+      - name: us-docker.pkg.dev/stackpulse/public/slack/message/snippet
+        id: slack-send-snippet
+        env:
+          CONTENT: >
```
Comment (hagaishapira) [inline aws/asg-execute-command/playbook.yaml:152]: give a meaningful name to the snippet

**Review by hagaishapira — CHANGES_REQUESTED** [review]
Please make the parameters view look more refine, use headings, use light mode like we've said. 
image

**Review by hagaishapira — APPROVED** [review]

**Pull request merged**